### PR TITLE
Add localization for the native part

### PR DESCRIPTION
**Pull request by smalluban**
## Summary

* SwiftUI is localized out-of-the-box
* For strings outside of the UI we use `LocalizedStringKey`
* For the HTML page we pass the localized messages by invoking JS code when the page is initalized

## Limitations

The Transifex integration is limited, as it does not support generated `xcloc` folders from the Xcode, but only the `.xliff` files, which are included inside. Because of that, the mechanism for updating the translated messages is only partly automatic (only pushing new messages to the server).

The `xcode/Shared (App)/StoreHelper/ViewModels/SubscriptionViewModel.swift` file generates the text on the fly by concatenation of words and messages, which is hard to integrate with the SwiftUI for automatic translation.

### CODE REVIEW

**Review by philipp-classen — APPROVED** [review]

File: xcode/Shared (App)/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (chrmod) [inline xcode/Shared (App)/en.lproj/Localizable.strings:1]: is this file needed?

File: package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "locales.wtm": "hybrids extract -c --format=chrome.i18n node_modules/@whotracksme ./src/_locales/en/messages.json",
     "locales.ui": "hybrids extract -c --format=chrome.i18n node_modules/@ghostery/ui ./src/_locales/en/messages.json",
     "locales.src": "hybrids extract -c --format=chrome.i18n ./src ./src/_locales/en/messages.json",
-    "locales": "npm run locales.wtm && npm run locales.ui && npm run locales.src"
+    "locales": "npm run locales.wtm && npm run locales.ui && npm run locales.src",
+    "xcode-export-locales": "rm -rf ./xcode/en.xcloc && xcodebuild -exportLocalizations -project ./xcode/Ghostery.xcodeproj -localizationPath ./xcode"
```
Comment (chrmod) [inline package.json:21]: would be nice to put all locale files into one folder like `./xcode/locales/`

**Review by chrmod — CHANGES_REQUESTED** [review]
Is it correct that LaunchScreen.storyboard` have moved to `en.lproj` ?

File: package.json (hunk 2 above)
Comment (smalluban) [reply package.json:21]: This path is in `.gitignore`, so it is only local structure needed for the extracting.

File: xcode/Shared (App)/en.lproj/Localizable.strings (hunk 1 above)
Comment (smalluban) [reply xcode/Shared (App)/en.lproj/Localizable.strings:1]: Yes, without that file, the translations with the export function generate duplicates for both projects (macOS, iOS). After the translation to other languages, this is a source file for the different versions.

**Comment by smalluban** [comment]
@chrmod ping ;)

**Review by chrmod — APPROVED** [review]
